common/src/subgraph_client/client.rs

```diff
@@ -189,7 +229,7 @@ impl SubgraphClient {
     pub async fn query<T: for<'de> Deserialize<'de>>(
         &self,
         query: impl IntoRequestParameters + Send + Clone,
-    ) -> Result<ResponseResult<T>, anyhow::Error> {
```

Comment:
Update `ResponseResult` to be `Result<T, String>` on the definition

Reply:
Perhaps you saw the old commit?
https://github.com/graphprotocol/indexer-rs/blob/a575b00262c03a34fb92759147fcac0896356892/common/src/subgraph_client/client.rs#L232

Reply:
nvm, I thought that `ResponseResult` was from our crate, but it's from thegraph-core